<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>friendlist</class>
 <widget class="QWidget" name="friendlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>16</y>
     <width>681</width>
     <height>273</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hor">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="show_Msg_text">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="friendlist_list">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="inputTX"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="delFriend_pb">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>删除好友</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="flush_pb">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>刷新好友</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showOnline_pb">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>显示在线用户</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="serchFriend_pb">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查找用户</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="chat_pb">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>私聊</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendMsg_pb">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>信息发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
